config/routes.rb

```diff
@@ -18,4 +18,12 @@
 
   resources :cart_items, only: [:create, :destroy]
   get "/cart", to: "cart_items#index"
+
+  get 'errors/file_not_found'
+  get 'errors/unprocessable'
+  get 'errors/internal_server_error'
+
+  match '/404', to: 'errors#file_not_found', via: :all
+  match '/422', to: 'errors#unprocessable', via: :all
+  match '/500', to: 'errors#internal_server_error', via: :all
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.